src/AspNetCoreModuleV2/CommonLib/FileOutputManager.cpp

```diff
@@ -0,0 +1,188 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+#include "stdafx.h"
+
+
+FileOutputManager::FileOutputManager()
+    : m_pStdOutFile(NULL)
+{
+}
+
+FileOutputManager::~FileOutputManager()
+{
+    HANDLE   handle = NULL;
+    WIN32_FIND_DATA fileData;
+
+    if (m_pStdOutFile != NULL)
```

Comment:
have you run appverifier to see whether there is an handle leak? 
I saw m_hLogFileHandle was not cleaned.

Reply:
https://msdn.microsoft.com/en-us/library/ks2530z6.aspx
> To close a file opened with _get_osfhandle, call _close. The underlying handle is also closed by a call to _close, so it is not necessary to call the Win32 function CloseHandle on the original handle.